packages/docs/src/scripts/components/Page.jsx

```diff
@@ -58,48 +55,46 @@ class Page extends React.PureComponent {
     return <PageBlock {...this.props} hideHeader />;
   }
 
+  renderGuidanceTabPanel() {
```

Comment:
Can we rename this to be `renderGuidanceSection`

Reply:
@line47 I think this is the last thing for me

Reply:
Updated 👍